### DISCUSSION TITLE
Consiglio acquisto sensore temperatura smart con display

### DISCUSSION THREAD

**Discussion by Foofighterclub** [indomus/forum#2750] [GENERALE - Discussioni generiche]
Buongiorno.
Stavo cercando un sensore temperatura wireless con display.
Avevo visto lo SwitchBot Meter Plus, il SONOFF SNZB-02D e lo Shelly Plus H&T.
Non avendo nessun Smart Hub, a parte Enel Homix Home, il quale però è compatibile solo con i suoi dispositivi e pochi altri accessori, ero più orientato verso lo Shelly, visto che è dotato di wifi, a meno che non sia un prodotto valido e siano preferibili altri dispositivi.
Avevo valutato anche lo SwitchBot, abbinandolo all'acquisto di un SwitchBot Hub 2, però visto anche il prezzo aggiuntivo da sborsare, non saprei se poi riuscirei a sfruttare anche l'acquisto dell'hub.

Grazie.

**Comment by 50494554524F** [comment]
il più piccolo ed economico è lo xiaomi thermometer 2 che trovi su aliexpress a meno di 5€ anche 4€ se capiti nel periodo di sconti... bluetooth ma arriva veramente lontano se il segnale non fa troppi angoli con gli ostacoli che si trova davanti, da moddare col firmware pvvx (gestisce anche bluetooth long range), non servono hub

se invece vuoi un display ampio, sempre xiaomi ne ha di più grandi ma più sottili di quelli che hai postato...

**Reply by Foofighterclub** [reply]
Ok, grazie.

**Reply by 50494554524F** [reply]
sui termometri xiaomi, tramite firmware pvvx, adesso si può usare anche zigbee

**Comment by KC-inDomus** [comment]
> Buongiorno. Stavo cercando un sensore temperatura wireless con display. Avevo visto lo SwitchBot Meter Plus, il SONOFF SNZB-02D e lo Shelly Plus H&T. Non avendo nessun Smart Hub, a parte Enel Homix Home, il quale però è compatibile solo con i suoi dispositivi e pochi altri accessori, ero più orientato verso lo Shelly, visto che è dotato di wifi, a meno che non sia un prodotto valido e siano preferibili altri dispositivi. Avevo valutato anche lo SwitchBot, abbinandolo all'acquisto di un SwitchBot Hub 2, però visto anche il prezzo aggiuntivo da sborsare, non saprei se poi riuscirei a sfruttare anche l'acquisto dell'hub.
> 
> Grazie.

la domada che devi farti è che intendi fare in futuro. perché se ti serve un solo senore e finisce lì, prendi la cosa meno costosa e senza hub, lo shelly h&t va più che bene.
diversamente fermati, e dotati di un hub personale. POI semmai pensa a cosa integrarvi sopra.
